Makefile

```diff
@@ -615,7 +622,7 @@ realclean: clean java_clean
 	@$(RM) -f *.bin debian/changelog*
 
 ### additionally removes generated files not in the repository tree
-distclean: realclean clean_genRules
+distclean: clean_stack realclean clean_genRules
```

Comment:
I'm still struggling to understand the debian build system/launchpad. I find the build system very confusing, but maybe it's only because of the docs.